server/test/unit/server/webservices/views/test_plugins.py

```diff
@@ -10,8 +10,56 @@
 
 
 class TestDistributorResourceView(unittest.TestCase):
-    pass
+    """
+    Tests for views for a single distributor.
+    """
+
+    @mock.patch('pulp.server.webservices.controllers.decorators._verify_auth',
+                new=assert_auth_READ())
+    @mock.patch('pulp.server.webservices.views.plugins.generate_json_response')
+    @mock.patch('pulp.server.webservices.views.plugins.factory')
+    def test_get_distributor_resource_with_existing_distributor(self, mock_factory, mock_serial):
+        """
+        DistributorResource should return a serialized object of the specified distributor.
+        """
+        mock_manager = mock.MagicMock()
+        mock_manager.distributors.return_value = [{'id': 'mock_distributor_1'},
+                                                  {'id': 'mock_distributor_2'}]
+        mock_factory.plugin_manager.return_value = mock_manager
+        request = mock.MagicMock()
+        request.get_full_path.return_value = '/mock/path/'
+
+        distributor_resource = DistributorResourceView()
+        distributor_resource.get(request, 'mock_distributor_2')
```

Comment:
Just to make sure the right thing gets returned, you could do this, assuming ``response`` is the value returned on line 33:

``self.assertTrue(response is mock_serial.return_value)``